specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/SessionPools_CreateOrUpdate.json

```diff
@@ -0,0 +1,140 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "sessionPoolName": "testsessionpool0",
```

Comment:
>testsessionpool0 [](http://example.com/codeflow?start=24&length=16)

should this be `testsessionpool` considering L50, L89, etc.

Reply:
Thanks, fixed